app/templates/share.js

```diff
@@ -10,20 +10,61 @@ function passwordComplete(state, password) {
   const el = html([
     `<div class="selectPassword">${state.translate('passwordResult', {
       password: '<pre></pre>'
-    })}</div>`
+    })}
+    <button id="resetButton">Reset</button>
+    <form id='reset-form' class="setPassword hidden" data-no-csrf>
+      <input id="unlock-reset-input"
+        class="unlock-input input-no-btn"
+        maxlength="64"
+        autocomplete="off"
+        placeholder="${state.translate('unlockInputPlaceholder')}">
+      <input type="submit"
+        id="unlock-reset-btn"
+        class="btn btn-hidden"
```

Comment:
Does it work if you set `onclick=${toggleResetInput}` here instead of with the querySelector below? If so it's the preferred style. (also same with others)

Reply:
Yes, I know that this is the preferred style, but I had tried to do it this way but it hadn't worked. I'll try again and if it doesn't work , I'll ask you for help :smiley:

Reply:
Nope , using `onclick=${toggleResetInput}` doesn't work :disappointed:. Can you try the same thing out once @dannycoates

Reply:
> We need to use the function form here so that the password text gets escaped properly. It's not a big deal.

So you mean we can still write `onclick=${toggleResetInput}`. Ok, I'll try searching more over the net of how to do this. Meanwhile if you can suggest me of how this is done, then I will surely rectify my mistake

Reply:
No, no, I was just commenting on why we needed to use html() instead of html``

Your code is fine :)